core/common/src/main/java/alluxio/exception/JobDoesNotExistRuntimeException.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.exception;
+
+import io.grpc.Status;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * The exception thrown when a job definition does not exist in Alluxio.
+ */
+@ThreadSafe
+public class JobDoesNotExistRuntimeException extends AlluxioRuntimeException {
```

Comment:
I suggest you use the BlockDoesNotExistException as the first example, if you are trying to find a starting point to move to the new Exception framework.